004-governance.Rmd

```diff
@@ -91,52 +93,52 @@ Authors are recruited from contributors.
 
 -   Set and clearly communicates the strategic objectives of the project.
 -   Oversee CRAN releases.
 -   On-board new authors.
--   Mediate conflicts amongst authors.
+-   Have the final say on important decisions.
+-   De-escalate conflict amongst authors and/or users.
 -   Enforce the code of conduct.
--   Recruit their replacement if they want to retire from the project.
+-   Recruit their replacement when they want to retire from the project.
 
 A maintainer has two special powers:
 
 -   They have **admin** access on Github, allowing them to add new authors when needed.
 
 -   They are listed in `Authors@R`, along with their email address and the "cre" (creator) role (a maintainer often isn't the creator, but CRAN mandates the use of this role).
 
-Maintainer turnover is slow and we have not yet developed a process for it, but we'd generally expect a maintainer to be a long-standing author.
-
 The maintainers of most of RStudio's open source repositories are RStudio employees.
 This is not a requirement but a recognition of the tension between making development open to all and ensuring that users can trust that a package will be maintained in the long term (i.e. 10+ years), which typically requires the maintainer be explicitly remunerated for their work.
 Where the maintainer is not an employee of RStudio, we ask for the "right of first refusal" — if the maintainer wants to stop maintaining the package (for whatever reason) they first offer it back to RStudio.
 In the future, we hope to find other ways of financial supporting maintainers apart from full-time employment.
 
-### Common tasks
+### Process
 
-#### Author invitation
+#### Invite author
 
-To on-board a new author, the maintainer looks for rough consensus amongst authors (which would include no objections within 7 days).
+To on-board a new author, the maintainer looks for rough consensus amongst authors by emailing them (since this needs to be done in private).
+(one signal of rough consensus would be no objections within 7 days).
 The maintainer then sends the following email:
 
 > Hi {name},
 >
-> In recognition of your significant contributions to {package}, would you be interested in becoming a co-author of the package?
-> This means that you'll be acknowledged in `Authors@R` and given write permission on GitHub.
-> Write permission gives you the power to change {package} directly (which is fine for smaller fixes), but I'd appreciate it if you'd continue to submit major changes through the pull request process.
+> In recognition of your significant contributions to {package}, would you be interested in becoming a package author?
+> You can read about the rights and responsibilities of a package author at ..., but in short, being an author means that you'll be acknowledged in `Authors@R` and given write permission on GitHub.
+> You'll continue to use PRs to contribute code, but merge your own PRs once they've been reviewed, and you can now review PRs from others.
```

Comment:
Oddly, I think basically anyone can review PRs. It's *requesting* review that is a privilege that requires higher-than-read permission. Along with other stuff like assigning, labelling, etc.

Reply:
Anyhow, maybe you just mean that morally an author should feel cool about reviewing, whereas it's more of a grey area for non-authors.

Reply:
Yeah, exactly.